row.py

```diff
@@ -924,6 +924,80 @@ def filter_ocr_results(original_results_file, out_dir):
     return out_dir
 
 
+def join_spreadsheet_info(filtered_results_file, out_dir):
+    """join additional information from the udot spreadsheets to the filtered ocr results
+
+    Args:
+        filtered_results_file (str): path to the csv file with filtered ocr results (path_to_file.csv)
+        out_dir (str): where to save the CSV file with filtered and joined results
+
+    Returns:
+        str: the location of the output CSV file with filtered and joined results
+    """
+    #: silence pandas SettingWithCopyWarning
+    pd.options.mode.chained_assignment = None
+
+    #: set up variables
+    spreadsheet_dir = location = Path(__file__).parent / "spreadsheets"
+    out_dir = Path(out_dir)
+
+    #: combine spreadsheets into a single df for joining
+    spreadsheets = list(spreadsheet_dir.glob("*.xlsx"))
```

Comment:
**quibble**: creating a list over the glob iterator is unnecessary. Remove the list call and assign to the spreadsheets variable or iterate over the result of the glob call.